### PR TITLE
feat(rdb): add computed attribute and upgrade node type

### PR DESCRIPTION
**Pull request by kindermoumoute**

### CODE REVIEW

File: scaleway/resource_rdb_instance_beta.go [hunk 1]
```diff
@@ -67,8 +68,53 @@ func resourceScalewayRdbInstanceBeta() *schema.Resource {
...
+						},
+					},
+				},
+			},
+			"certificate": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Certificate of the database instance",
+			},
+			"backup_schedule_frequency": {
```
Comment (jerome-quere) [inline scaleway/resource_rdb_instance_beta.go]: I don't think this computed is usefull

File: scaleway/resource_rdb_instance_beta.go [hunk 2]
```diff
@@ -140,9 +185,29 @@ func resourceScalewayRdbInstanceBetaRead(d *schema.ResourceData, m interface{})
 	d.Set("user_name", d.Get("user_name").(string)) // user name and
 	d.Set("password", d.Get("password").(string))   // password are immutable
 	d.Set("tags", res.Tags)
+	if res.Endpoint != nil {
```
Comment (jerome-quere) [inline scaleway/resource_rdb_instance_beta.go]: If res.Endpoint is null you should set the value to empty string.

File: website/docs/r/rdb_instance_beta.html.markdown [hunk 3]
```diff
@@ -33,7 +33,7 @@ The following arguments are supported:
 
 - `node_type` - (Required) The type of database instance you want to create (e.g. `db-dev-s`).
 
-~> **Important:** Updates to `node_type` will recreate the Database Instance.
+~> **Important:** Updates to `node_type` will upgrade the Database Instance to the desired `node_type` without any interruption.
```
Comment (jerome-quere) [inline website/docs/r/rdb_instance_beta.html.markdown]: Add a note that you cannot downgrade and instance

File: scaleway/resource_rdb_instance_beta.go (hunk 2 above)
Comment (kindermoumoute) [reply scaleway/resource_rdb_instance_beta.go]: Should I set `endpoint_port` value to zero as well? 🤔

Comment (jerome-quere) [reply scaleway/resource_rdb_instance_beta.go]: yes

**Pull request merged**